pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -72,6 +67,12 @@ in buildPythonPackage rec {
   PYTORCH_BUILD_VERSION = version;
   PYTORCH_BUILD_NUMBER = 0;
 
+  USE_FBGEMM = 0; # this can't build because of CMAKE downloads
+  USE_MKLDNN = 0; # mkl with cuda is broken
+  USE_NCCL = 0; # multigpu looks broken broken
```

Comment:
Did you try including [NCCL](https://github.com/NixOS/nixpkgs/blob/5d86f9a9ce86be51f20d213a43e96f8694f08169/pkgs/development/libraries/science/math/nccl/default.nix)?

Reply:
Yes, unfortunately.

Reply:
Maybe this is because nvlink is attempting to perform checks in the GPU runtime?